src/app/tokens.ts

```diff
@@ -0,0 +1,92 @@
+import { Token } from "./base";
+import { AppComponent } from "./app.component";
+
```

Comment:
Again, provide comments about what these constants represent, how they should be used, and how to add constants to this list. Think about dev 5 years from now.